### ISSUE TITLE
Incorrect opportunity type in ERP Next not in English

### ISSUE THREAD

**Issue by jaimestuardo** [frappe/erpnext#23432]
## Description

When opportunity comes from a Lead, opportunity type is incorrect when the system is not in English. This problem exists since version 11.

## Context information (for bug reports)

**Output of `bench version`**
```
ERPNext: v12.11.2 (version-12)

Frappe Framework: v12.9.1 (version-12)
```

## Steps to reproduce the issue

1. Set ERPNext in other language, for example, Spanish
2. Create a Lead
3. Click on `Make` button and choose `Opportunity`

### Observed result

Opportunity page will open. In `Opportunity type` field will appear: `Sales`

After clicking on `Save` button, an alert telling that opportunity type is incorrect. To fix it, one has to manually change `Sales` by `Ventas` in `Opportunity type` field.

### Expected result

After making the opportunity from lead, `Ventas` should be selected by default, so, the selected element should be according to the language of the system.

### Stacktrace / full error message

```
N/A
```

## Additional information

`ERPNext` was installed manually, using `bench init erpnext && cd erpnext` and then calling `bench --site erp.example.com install-app erpnext`

**Comment by p010x** [comment]
Still not resolved in v13.0.0 
+1

**Comment by vishdha** [comment]
https://github.com/frappe/frappe/pull/17591
